test/parallel/test-os.js

```diff
@@ -85,9 +85,12 @@ const hostname = os.hostname();
 is.string(hostname);
 assert.ok(hostname.length > 0);
 
-const uptime = os.uptime();
-is.number(uptime);
-assert.ok(uptime > 0);
+// On IBMi, os.uptime() returns 'undefined'
+if (!common.isIBMi) {
+  const uptime = os.uptime();
+  is.number(uptime);
+  assert.ok(uptime > 0);
+}
```

Comment:
Should we maybe explicitly test IBMi as well and verify that it is indeed always undefined?

Reply:
Hello @BridgeAR , I tried below code (copied from [the libuv aix.c](https://github.com/nodejs/node/blob/master/deps/uv/src/unix/aix.c#L959-L983)) on IBMi, the uptime is not correct obviously -- 
```
#include <stdio.h>
#include <sys/types.h>
#include <utmp.h>
#include <errno.h>

int main(double *uptime)
{
    struct utmp *utmp_buf;
    size_t entries = 0;
    time_t now_time, boot_time;
    utmpname(UTMP_FILE);
    setutent();
    while ((utmp_buf = getutent()) != NULL)
    {
        if (utmp_buf->ut_user[0] && utmp_buf->ut_type == USER_PROCESS)
            ++entries;
        if (utmp_buf->ut_type == BOOT_TIME)
            boot_time = utmp_buf->ut_time;
    }
    endutent();
    if (boot_time == 0)
        return -ENOSYS;
    now_time = time(NULL);
    printf("now_time = %ld, boot_time = %ld, uptime = %ld\n", now_time, boot_time, now_time - boot_time);
    return 0;
}
```
Result:
```
now_time = 1575810624, boot_time = -4982089500409860083, uptime = 4982089501985670707
```
We still do not find a way to get uptime on IBMi by now.